data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg

```diff
@@ -289,13 +289,17 @@
         [if]
             [have_unit]
                 side=1
-                x=$starting_point.x
-                y=$starting_point.y
-                range=10
-                [not]
+                [filter_location]
```

Comment:
this coudl probaly be impoved to use location_id=1

Reply:
What would that do? Looks like that feature is still undocumented, anyway.

Reply:
`location_id=number` means "the starting location of that side".